### PR TITLE
Remove gogoproto options

### PR DESCRIPTION
**Pull request by johanstokking**
#### Summary

References #2798

#### Changes

Remove gogoproto options from `lorawan.proto`, `regional.proto`, `qrcodegenerator.proto` and `messages.proto`.

#### Testing

CI

##### Regressions

None expected, will do some additional testing

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: cmd/ttn-lw-cli/commands/lorawan.go [hunk 1]
```diff
@@ -165,11 +165,11 @@ func decodeDownlink(msg ttnpb.Message, config lorawanConfig) (*lorawanDecodedFra
...
 
 	macBuf := getMacBuffer(pld)
-	if len(macBuf) > 0 && (len(pld.FOpts) == 0 || config.MACVersion.EncryptFOpts()) && !config.NwkSKey.IsZero() {
+	if len(macBuf) > 0 && (len(pld.FHdr.FOpts) == 0 || config.MACVersion.EncryptFOpts()) && !config.NwkSKey.IsZero() {
 		logger.Debug("Decrypting MAC buffer")
-		for msb := uint32(0); msb < 0xff; msb++ {
-			fCnt := msb<<8 | pld.FCnt
-			macBuf, err := crypto.DecryptDownlink(config.NwkSKey, pld.DevAddr, fCnt, macBuf, pld.FPort != 0)
+		for msb := uint32(0); msb < 0xffff; msb++ {
+			fCnt := msb<<16 | pld.FHdr.FCnt
```
Comment (johanstokking) [inline cmd/ttn-lw-cli/commands/lorawan.go:171]: Unrelated bugfix

File: cmd/ttn-lw-cli/commands/simulate.go [hunk 2]
```diff
@@ -693,9 +693,8 @@ var (
 			if err != nil {
 				return err
 			}
-			t := time.Now().UTC()
-			if uplinkMessage.ReceivedAt.IsZero() {
-				uplinkMessage.ReceivedAt = &t
+			if uplinkMessage.ReceivedAt == nil {
+				uplinkMessage.ReceivedAt = ttnpb.ProtoTimePtr(time.Now().UTC())
```
Comment (adriansmares) [inline cmd/ttn-lw-cli/commands/simulate.go]: `ProtoTimePtr` doesn't need `.UTC()`.

File: pkg/interop/client.go [hunk 3]
```diff
@@ -60,9 +60,10 @@ type jsRPCPaths struct {
 }
 
 func (p jsRPCPaths) join() string    { return p.Join }
-func (p jsRPCPaths) rejoin() string  { return p.Rejoin }
 func (p jsRPCPaths) appSKey() string { return p.AppSKey }
-func (p jsRPCPaths) homeNS() string  { return p.HomeNS }
+
+//func (p jsRPCPaths) rejoin() string  { return p.Rejoin }
```
Comment (adriansmares) [inline pkg/interop/client.go]: Can be removed ?

**Review by adriansmares — APPROVED** [review]
LGTM. My only comment to what I see is that there is no point in converting to UTC, and then to the `pbtypes` timestamps, because the `pbtypes` timestamps are location-agnostic anyway (Unix time).

File: cmd/ttn-lw-cli/commands/simulate.go (hunk 2 above)
Comment (johanstokking) [reply cmd/ttn-lw-cli/commands/simulate.go]: https://github.com/TheThingsNetwork/lorawan-stack/pull/4917/commits/ef3569b69185d2cf860932a8c17d71953d4c96e1

File: pkg/interop/client.go (hunk 3 above)
Comment (johanstokking) [reply pkg/interop/client.go]: https://github.com/TheThingsNetwork/lorawan-stack/pull/4917/commits/3f51ddc999b8cf21b8580b65ebecf1112b9b5808

**Pull request merged**